common/src/main/scala/explore/GraphQLSchemas.scala

```diff
@@ -0,0 +1,25 @@
+package explore
+
+import clue.macros.GraphQLSchema
+import explore.model.SiderealTarget
+import explore.model.enum._
+
+object GraphQLSchemas {
+  @GraphQLSchema(debug = false)
```

Comment:
You get the compiler to show you the generated code. In case you need to know what's going on, you just switch to `true`, and in VSCode at least you see this:

![image](https://user-images.githubusercontent.com/1895643/94930811-59543700-049d-11eb-8850-92df03ed6986.png)

It applies to `@GraphQL` too. We probably won't be using this much anyway, since metals gives you code completions with all the generated types/methods.